lib/trufi_app.dart

```diff
@@ -145,14 +144,10 @@ class _LocalizedMaterialAppState extends State<LocalizedMaterialApp> {
             );
           },
           localizationsDelegates: [
-            TrufiLocalizationsDelegate(
-              snapshot.data,
-              widget.localization,
```

Comment:
This is the only area I am unsure of, it seems that we were able to overwrite the location from the host system. This would not be possible anymore and they needed to create the arb file and generate it inside of Trufi_Core.

Reply:
We discussed that we provide custom translations for specific keys to the host app.